fast/diffusion_2D_wBCs.py

```diff
@@ -27,6 +26,9 @@
 parser.add_argument("-xdsl", "--xdsl", default=False, type=bool, help="xDSL run")
 args = parser.parse_args()
 
+if args.plot:
+    from fast.bench_utils import plot_2dfunc
```

Comment:
well, importing one func only is very cheap to add it this way, leave it as it was before I would say

Reply:
Can do, it's not really related
Just to explain, I was trying to avoid the Matplotlib font cache building overhead on ARCHER2/Cirrus, where I wouldn't say it's cheap